net/https-dns-proxy/files/README.md

```diff
@@ -1,5 +1,8 @@
+<!-- markdownlint-disable MD013 -->
 # DNS Over HTTPS Proxy (https-dns-proxy)
 
+[![HitCount](http://hits.dwyl.com/stangri/openwrt/https-dns-proxy.svg)](http://hits.dwyl.com/stangri/openwrt/https-dns-proxy)
```

Comment:
No way. Sorry. NACK for this change.

Reply:
Yeah, please explain why?

Reply:
User tracking without user consent - not very trustworthy ...

Reply:
> User tracking without user consent - not very trustworthy ...

By tracking you mean placing a cookie? My understanding was it's the same as counting hits on a screenshot, but I see that there is a hits.dwyl.com cooking in my browser, so I guess I was wrong on that.

Just to clarify, there's nothing against the cookie-less counter of visits, right? Like as if I count hits on the screenshot and display the counter without the cookie?

Reply:
In my opinion it's not a technical question regarding cookies, hidden pixels, analytics, tracking infrastructure etc., simply take the user privacy seriously and don't collect any data without consent. There is no valid reason for a piece of software within a opensource repo to ship any kind of data collector. 
In europe the GDPR (General Data Protection Regulation) has six legal bases outlined for processing personal data (https://gdpr.eu/article-6-how-to-process-personal-data-legally/) and none of them matched.

Bottom line: just don't do it - please.

Reply:
> In my opinion it's not a technical question regarding cookies, hidden pixels, analytics, tracking infrastructure etc., simply take the user privacy seriously and don't collect any data without consent. There is no valid reason for a piece of software within a opensource repo to ship any kind of data collector. 

I totally agree. No personal data is collected.

> In europe the GDPR (General Data Protection Regulation) has six legal bases outlined for processing personal data (https://gdpr.eu/article-6-how-to-process-personal-data-legally/) and none of them matched.

Again, no personal data is collected.


> Bottom line: just don't do it - please.

I don't. Moreover this is the github-targeting counter, so almost all if not all its users are open source projects/developers. I've reached out to the author and they confirmed that even a cookie doesn't come from the counter but from the cloudlfare as with any cloudlfare-protected site.

So I'm yet again asking why the knee-jerk reaction to the counter which doesn't collect any personal information?

I'm not opposed to taking the readme to the personal repo if the hit counter is the reason it can't be accepted to packages repo, I just find it puzzling.

Reply:
Please do not close your eyes to the obvious: Look in the issue list of the referenced counter repo (marked as "enhancements"): "How to Track and Display Profile Views on GitHub", "Analytics for Startups", "Create a context for the user, useragent and repository database calls" and so on ... counters were popular in the 1990s, but were later replaced by other web traffic measures such as self-hosted scripts like Analog, and later on by remote systems that used JavaScript, like Google Analytics. Don't collect any data that you do not need!

You have two NACKs for this change.

Reply:
> Don't collect any data that you do not need!

Just to be clear, you agree that no personal data is collected, you're just opposed to the counter itself?

Reply:
README has been moved out of OpenWrt repo.